monitor/src/main.c

```diff
@@ -79,7 +79,7 @@
  *
  * FIXME: This can be smaller once compression is enabled.
  */
-#define BOOTSTRAP_INVOCATION_DATA_SIZE 110
+#define BOOTSTRAP_INVOCATION_DATA_SIZE 256
```

Comment:
I think the comment could also get a few more words then, where the magic value roughly comes from.

Reply:
If this part of the monitor was sticking around it would be part of https://github.com/seL4/microkit/issues/49 but it's going to be removed in the transition to capDL so I think we'll leave the comment as it is for now.